.gitignore

```diff
@@ -0,0 +1,152 @@
+# Created by .ignore support plugin (hsz.mobi)
+### Python template
+# Byte-compiled / optimized / DLL files
+__pycache__/
```

Comment:
The gitignore or changes in it shouldnt be a part of the pull request, it should be commits directly in main in the initial commit, or merged afterwards, without a pull request.